### DISCUSSION TITLE
Plans for future Iran: How to rebuild it

### DISCUSSION THREAD

**Discussion by Homap** [WeAreMahsaAmini/Core#320] [Ideas]
Hi all,

I suggest to make a list for plans for the future Iran. This is both practical and hopeful for the young people in Iran, to know we believe in them and we are with them. How are we going to rebuild Iran? It could be a list of areas, such as climate, wildlife, justice system, infrastructure, tourism, etc etc and then when clicking on each list, we have a list of ideas for future, all based on a democratic system where everyone gets to be heard.

**Comment by Homap** [comment]
Can I start a repository for this? 

It would be wonderful to have this. We can see and envision the future.

**Reply by atenajoon** [reply]
great idea, but I would put current issues on priority tickets.

**Reply by Homap** [reply]
You are right. I wonder where I can help. I can code in Python, Bash and R and analyse data but I am a scientist, have only ever worked with research questions. I'm not sure how I can use my skills in our current projects. I am trying to see where I can be useful but please let me know if I can help.
